docs/contributing/contributing-icons.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to the Icons
+
+We're glad to know you've decided to contribute!
+You can get started by forking the repo or cloning it locally.
+
+Click on the "Fork" button at the top of the repository and that will take you to the fork creation page. You can select the owner, repo name etc there and create your fork.
+Alternatively, you can start making changes to the original repo on the GitHub website and that will automatically create a fork and save changes to it.
+
+You can run the following command to clone the forked repo locally (You must have [Git](https://git-scm.com/)):
+
+```
+git clone https://github.com/<your-github-username>/LinkFree
+```
+
+Replace `<your-github-username>` with your GitHub username.
+
+## How to add icons
+
+To add a new icon, you need to modify the following files: [`GetIcons.js`](https://github.com/EddieHubCommunity/LinkFree/blob/main/src/Components/Icons/GetIcons.js), [`icons.md`](https://github.com/EddieHubCommunity/LinkFree/blob/main/icons.md), and [`links.json`](https://github.com/EddieHubCommunity/LinkFree/blob/main/src/config/links.json).
+
+The `GetIcons.js` file contains the main code that shows the icons on the page. It has a `GetIcons` component which takes a `iconName` prop and returns an icon according to that. You need to add another case inside the switch statement in the component for the new icon.
+
+The `icons.md` file is the index of all available icons. If you add a new icon, be sure to update this index so that people can see what's the code of the icon and how does it look like.
+
+The `links.json` file has `validIcons` object which specifies the color codes for the icons. Be sure to add the correct color for the new icon!
```

Comment:
Hmm, maybe we could add an example of adding a new icon? For example: adding the Spotify icon, or any other icon (it can be in the list, too, just needs to be an example). What do you think? 🙂

Reply:
I've added that!